audio/Kconfig

```diff
@@ -104,6 +104,12 @@ config AUDIO_FORMAT_MP3
 	---help---
 		Build in support for MP3 Audio format.
 
+config AUDIO_FORMAT_SBC
+        bool "Bluetooth SubBand Coding"
+        default y
+        ---help---
```

Comment:
@shipei-xiaomi should this config option "depends on WIRELESS_BLUETOOTH" ?

Reply:
Not necessary, because https://github.com/apache/nuttx-apps/pull/1887 could play SBC file without bluetooth.